packages/eslint-plugin/tests/rules/ban-ts-comment.test.ts

```diff
@@ -228,6 +228,22 @@ if (false) {
         },
       ],
     },
+    {
+      code: noFormat`// @ts-expect-error 👨‍👩‍👧‍👦`,
```

Comment:
The new added `invalid` test cases look great. Could you also add a `valid` test case too?

https://github.com/eslint/eslint/pull/16154 and https://github.com/eslint/eslint/pull/16321 mention including more test cases around other unusual characters. I don't think we particularly need them here (the package is well-tested).

Reply:
Added valid test cases: [3d53073](https://github.com/typescript-eslint/typescript-eslint/pull/5704/commits/3d5307361405178205931e5066adab7491f9a2cc)